src/component/chrome/overview-switch-container.tsx

```diff
@@ -0,0 +1,32 @@
+// import * as MobX from 'mobx';
```

Comment:
if you don't need this one please delete the line

Reply:
sorry, my bad. Done too :)